### PR TITLE
Ext for gql api

### PR DESCRIPTION
**Pull request by akshbhu**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-graphql-transformer-core/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
         "@aws-cdk/aws-dynamodb": "~1.124.0",
         "@aws-cdk/aws-ec2": "~1.124.0",
         "@aws-cdk/aws-efs": "~1.124.0",
+        "@aws-cdk/aws-elasticsearch": "^1.129.0",
```
Comment (jhockett) [inline packages/amplify-graphql-transformer-core/package.json:41]:
```suggestion
        "@aws-cdk/aws-elasticsearch": "~1.124.0",
```

**Review by jhockett — CHANGES_REQUESTED** [review]

File: packages/amplify-graphql-transformer-core/src/transformation/transform.ts [hunk 2]
```diff
@@ -250,9 +264,113 @@ export class GraphQLTransform {
...
+              } else {
+                amplifyApiObj[fieldType][fieldName] = {};
+                amplifyApiObj[fieldType][fieldName][resourceName] = resource;
+              }
+            }
+          }
+        }
+      }
+    });
+    // console.log(amplifyApiObj)
```
Comment (jhockett) [inline packages/amplify-graphql-transformer-core/src/transformation/transform.ts:346]: nit: remove

File: packages/amplify-graphql-transformer-core/src/transformation/transform.ts [hunk 3]
```diff
@@ -250,9 +264,113 @@ export class GraphQLTransform {
...
+      } catch (err) {
+        const error = new Error(`Skipping override due to ${err}${os.EOL}`);
+        printer.error(`${error}`);
+        error.stack = undefined;
+        throw error;
+      }
+    }
+    // appsyncResourceObj.api!.GraphQLAPI!.xrayEnabled = true;
+    // appsyncResourceObj.models!['Todo']!.modelDDBTable!.billingMode = 'PROVISIONED';
+    // console.log(appsyncResourceObj);
```
Comment (jhockett) [inline packages/amplify-graphql-transformer-core/src/transformation/transform.ts:371]: nit: remove if not needed

File: packages/amplify-graphql-transformer-core/src/transformation/transform.ts [hunk 4]
```diff
@@ -250,9 +264,113 @@ export class GraphQLTransform {
...
+        },
+      });
+      try {
+        sandboxNode.run(overrideCode, path.join(this.overrideConfig!.overrideDir, 'build', 'override.js')).override(appsyncResourceObj);
+      } catch (err) {
+        const error = new Error(`Skipping override due to ${err}${os.EOL}`);
+        printer.error(`${error}`);
+        error.stack = undefined;
+        throw error;
+      }
```
Comment (jhockett) [inline packages/amplify-graphql-transformer-core/src/transformation/transform.ts:367]: Can this logic be generalized and moved to `amplify-cli-core`?

Comment (akshbhu) [reply packages/amplify-graphql-transformer-core/src/transformation/transform.ts:367]: this package is indepedent of cli-core

File: packages/amplify-category-api/resources/schemas/appsync/AppsyncCLIInputs.schema.json [hunk 5]
```diff
@@ -0,0 +1,289 @@
+{
```
Comment (jhockett) [inline packages/amplify-category-api/resources/schemas/appsync/AppsyncCLIInputs.schema.json:1]: This file looks like it was renamed. Please remove this one that doesn't seem relevant anymore

**Review by jhockett — APPROVED** [review]

**Pull request merged**